org.lflang/src/org/lflang/LinguaFranca.xtext

```diff
@@ -37,7 +37,10 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * @author{Edward A. Lee <eal@berkeley.edu>}
  */
 
-grammar org.lflang.LF with org.eclipse.xtext.common.Terminals
```

Comment:
I don't really understand why this change was necessary. Is it? Doesn't the override mechanism suffice?

Reply:
I don't think it's necessary, but I find it clearer to have all terminals be defined in the same place. Otherwise it's not so easy to determine how the lexer behaves. I also was editing the grammar outside of eclipse so looking up the external file was a pain.

We were anyway overriding all of them except INT, ID and WS

Reply:
OK, fair enough :-)